x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/policy_response.cy.ts

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Got curious, why is this in the `mocked_data` folder?

Reply:
there was no need to actually have a real endpoint for this test, thus I did not include it in the other endpoint suite. Also, the "real endpoint" test suite does not run in CI (yet 🤞 )